helm-chart/values.yaml

```diff
@@ -1,11 +1,28 @@
 ---
-releaseImageTag: cf66cfb # Need to pass this value when applying the helm chart (This is the $BUILD_ID )
+# Need to pass $BUILD_ID to this key when applying the helm chart in CI. 
+# or 
+# Set the commit hash for a particular buiild to deploy built image for that commit ID. (NOTE: This would work only if the image is built for that commit ID.)
+releaseImageTag: cf66cfb 
+
+# Setting this to true will make this a sandbox environment. It'll set the following:
+# - nodeAffinity for the Monitor, Nodejs, Redis, Varnish, Web & Nginx pod to schedule opn node where preemptible=true
+# - Set appropriate labels to Nginx pod
+# - Requests images with the tag "sandbox-<releaseImageTag>"
 sandbox: "false"
-sandboxName: ""
-deployEnv: "deadbeef" # This value should not appear in finalized templates # Need to pass this value when applying the helm chart
+# Setting this to true setsup liveness, readiness & startup probes on the web pod. 
 contentSandbox: "false"
+
+# This value sets the name of the environment and it's associated objects. Some suggestions for the values are prod/dev/test
+deployEnv: "dev" 
+
+# Helps create services for nginx, nodeja, varnish and web pods with appropriate tags that help ArgoCD do blue green deployments.
 previousServicesCount: "3"
-projectID: development-205018 # Need to pass this value when applying the helm chart
+
+# This is the project ID for images coming from gcr. Ther are two possible values for this currently development-205018 / production-deployment.
+# Should pass ${PROJECT_ID} value to this when applying the helm chart in the CI pipeline.
+projectID: development-205018 
```

Comment:
This shouldn't be defaulted

Reply:
and in fact we shouldn't need it at all, we should have a single variable called `imageRegistry` which would be the full path to either the gcr project or docker etc..

Reply:
I am just wondering how we'd handle the different projectIDs that the images are being release under. Do we expect people to change the value for `imageRegistry` for their helm release? Is it understood that everyone using the helm will have the visibility of the available or possible registry names?

Reply:
There are 2 use cases - the general public who will not have access to GCP - only to a public registry e.g. dockerhub or github packages, and the other to people who have access GCP. 

The default should be publicly available, and since there are none at this point, it should default to empty - We also shouldn't be specifying the project id at all, rather just the imageRegistry prefix

Reply:
Noted.